src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,45 @@
+package lotto.view;
+
+import com.google.common.primitives.Ints;
+import lotto.domain.Lotto;
+import lotto.domain.Result;
+import lotto.domain.Statistics;
+
+import java.util.List;
+
+public class OutputView {
+
+    private OutputView() {
+    }
+
+    public static void showBuyLotto(List<Lotto> lottos) {
+        System.out.printf("%d개를 구매했습니다.\n", lottos.size());
+        StringBuilder sb = new StringBuilder();
+        for (Lotto lotto : lottos) {
+            sb.append("[");
+            sb.append(Ints.join(", ", lotto.getLottoNumbers().stream().mapToInt(i -> i).toArray()));
+            sb.append("]\n");
+        }
+        System.out.println(sb.toString());
+    }
+
+    public static void result(List<Result> values,Statistics statistics) {
+        for (Result result : values) {
+            System.out.printf("%d개 일치%s(%d원)- %d개\n",
+                    result.getCount(),
+                    result.getBonus(),
+                    result.getPrize(),
+                    statistics.getStatic(result)
+            );
+        }
+    }
```

Comment:
아래와 같은 방식을 사용하면 정렬된 List<Result>를 받지 않아도 될거 같습니다
```suggestion
    public static void result(Statistics statistics) {
        Arrays.stream(Result.values())
            .sorted((result1, result2) -> result2.getPrize() - result1.getPrize())
            .forEach(result ->
                    System.out.printf(
                            "%d개 일치%s(%d원)- %d개\n",
                            result.getCount(),
                            result.getBonus(),
                            result.getPrize(),
                            statistics.getStatic(result)
                    )
            );
    }
```